manifests/params.pp

```diff
@@ -392,7 +392,7 @@
         'wsgi'                  => 'libapache2-mod-wsgi',
         'xsendfile'             => 'libapache2-mod-xsendfile',
       }
-    } elsif ($::operatingsystem == 'Ubuntu') or ($::operatingsystem == 'Debian' and versioncmp($::operatingsystemrelease, '11') < 0) {
+    } elsif ($::operatingsystem == 'Ubuntu') or ($::operatingsystem == 'Debian' and versioncmp($facts['os']['release']['major'], '11') < 0) {
```

Comment:
I'd prefer to remain consistent in fact usage:
```suggestion
    } elsif ($::operatingsystem == 'Ubuntu') or ($::operatingsystem == 'Debian' and versioncmp($::operatingsystemmajrelease, '11') < 0) {
```

While not as pretty, it makes the diff very easy.